db/migrate/20220429092052_create_order_cycles_shipping_methods.rb

```diff
@@ -1,17 +1,16 @@
 class CreateOrderCycleShippingMethods < ActiveRecord::Migration[6.1]
   def up
-    create_table :order_cycle_shipping_methods do |t|
+    create_table :order_cycles_shipping_methods, id: false do |t|
       t.references :order_cycle
       t.references :shipping_method, foreign_key: { to_table: :spree_shipping_methods }
-      t.timestamps
     end
-    add_index :order_cycle_shipping_methods,
+    add_index :order_cycles_shipping_methods,
```

Comment:
Do we actually need this index? When do we query if a shipping method is associated to an order cycle?